client/src/components/Header/components/nav-links.tsx

```diff
@@ -334,247 +313,228 @@ class NavLinks extends Component<NavLinksProps, NavlinkStates> {
+        >
+          <span>{t('buttons.forum')}</span>
+          <FontAwesomeIcon icon={faExternalLinkAlt} />
+        </Link>
+      </li>
+      <li key='news'>
+        <Link
+          className='nav-link nav-link-flex'
+          external={true}
+          onKeyDown={handleMenuKeyDown}
+          sameTab={false}
+          to={t('links:nav.news')}
+        >
+          <span>{t('buttons.news')}</span>
+          <FontAwesomeIcon icon={faExternalLinkAlt} />
+        </Link>
+      </li>
+      <li key='radio'>
+        <Link
+          className='nav-link nav-link-flex'
+          external={true}
+          onKeyDown={handleMenuKeyDown}
+          sameTab={false}
+          to={radioLocation}
+        >
+          <span>{t('buttons.radio')}</span>
+          <FontAwesomeIcon icon={faExternalLinkAlt} />
+        </Link>
+      </li>
+      <li className='nav-line' key='theme'>
+        <button
+          {...(!currentUserName && { 'aria-describedby': 'theme-sign-in' })}
+          aria-disabled={!currentUserName}
+          aria-pressed={currentUserTheme === Themes.Night ? 'true' : 'false'}
+          className={
+            'nav-link nav-link-flex' +
+            (!currentUserName ? ' nav-link-header' : '')
+          }
+          onClick={() => {
+            if (currentUserName) {
+              toggleTheme(currentUserTheme, toggleNightMode);
             }
-            onClick={() => {
-              if (currentUserName) {
-                this.toggleTheme(currentUserTheme, toggleNightMode);
-              }
-            }}
-            onKeyDown={this.handleMenuKeyDown}
-          >
-            {currentUserName ? (
-              <>
-                <span>{t('settings.labels.night-mode')}</span>
-                {currentUserTheme === Themes.Night ? (
-                  <FontAwesomeIcon icon={faCheckSquare} />
-                ) : (
-                  <FontAwesomeIcon icon={faSquare} />
-                )}
-              </>
-            ) : (
-              <Fragment key='night-mode'>
-                <span className='sr-only'>
-                  {t('settings.labels.night-mode')}
-                </span>
-                <span
-                  aria-hidden='true'
-                  className='nav-link-dull'
-                  id='theme-sign-in'
-                >
-                  {t('misc.change-theme')}
-                </span>
-              </Fragment>
-            )}
-          </button>
-        </li>
-        <li key='lang-menu'>
-          {/* 
+          }}
+          onKeyDown={handleMenuKeyDown}
+        >
+          {currentUserName ? (
+            <>
+              <span>{t('settings.labels.night-mode')}</span>
+              {currentUserTheme === Themes.Night ? (
+                <FontAwesomeIcon icon={faCheckSquare} />
+              ) : (
+                <FontAwesomeIcon icon={faSquare} />
+              )}
+            </>
+          ) : (
+            <Fragment key='night-mode'>
```

Comment:
Why is this `key` here? What does it do?

Reply:
no idea =)

Reply:
Resolved to merge this, I will open PR removing this, and we can test this there